po/it.po

```diff
@@ -0,0 +1,1724 @@
+# ITALIAN TRANSLATION FOR FLY-PIE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
```

Comment:
You might want to add this additional info ;)

```suggestion
# Copyright (C) 2020 Simon Schneegans
# This file is distributed under the same license as the Fly-Pie package.
```

Reply:
> You might want to add this additional info ;)

Done!